packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/index.stories.ts

```diff
@@ -25,6 +24,10 @@ const defaultVariables = {
   preHeader: 'Random headers',
   privacyUrl: '',
   supportUrl: '',
+  headerText: "Here's your reminder to sync devices.",
+  bodyText:
+    'It takes two to sync. Syncing another device with Firefox privately keeps your bookmarks, passwords and other Firefox data the same everywhere you use Firefox.',
```

Comment:
How come we are setting this header and body text as variables and not directly in the template?

Reply:
In the mustache templates [`headerText`](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/templates/partials/cadReminder.html#L4) and other text content is being passed down from the `message` object. I wasn't sure of the best way to do the same with our storybook setup so passed down those variables from our stories file.